packages/fixie-common/src/agent.ts

```diff
@@ -301,11 +301,7 @@ export class FixieAgentBase {
     this.update({ currentRevisionId: revisionId });
   }
 
-  public deleteRevision(revisionId: string): Promise<void> {
-    return this.client.requestJson(
-      `/api/v1/agents/${this.metadata.agentId}/revisions/${revisionId}`,
-      undefined,
-      'DELETE'
-    );
+  public deleteRevision(revisionId: string) {
+    this.client.requestJson(`/api/v1/agents/${this.metadata.agentId}/revisions/${revisionId}`, undefined, 'DELETE');
```

Comment:
- Should either `return` or `await` the promise
- Should this just be calling `this.client.request`? Seems weird to invoke `requestJson` when there's no JSON

Reply:
Yeah, I'll clean this up everywhere.